recipes/catch2/2.x.x/CMakeLists.txt

```diff
@@ -7,10 +7,4 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 include(conanbuildinfo.cmake)
```

Comment:
This file should be deleted 🤔

Reply:
whoops, sloppy work by me sorry about that :)  have been under the weather a bit